File: Order.c
#include "Order.h"


struct order *create_order(int student_id,const char *student_name,int lib_id,const char *lib_name,const char *title,const char *author,int copies){
 struct order *newnode=(struct order *)malloc(sizeof(struct order));
    if(newnode == NULL){
        return NULL;
    }
    strncpy(newnode->author,author,sizeof(newnode->author)-1);//author
    strncpy(newnode->lib_name,lib_name,sizeof(lib_name)-1);//lib
    newnode->student_id=student_id;
    newnode->lib_id=lib_id;
    newnode->copies=copies;
    newnode->
}

struct order *insert_order(struct order *newnode,struct order *root){

File: smart_library_java/src/smartlibrary/Order.java
package smartlibrary;

public final class Order {
    public int studentId;
    public String studentName;
    public int libraryId;
    public String libraryName;
    public String title;
    public String author;
    public int copies;
    public int price;
    public final String[] status = {"Ordered-Placed", "In-Transit", "Out-For-Delivery"};

    public Order(int studentId, String studentName, int libraryId, String libraryName, String title, String author, int copies, int price) {
        this.studentId = studentId;
        this.studentName = studentName;
        this.libraryId = libraryId;
        this.libraryName = libraryName;
        this.title = title;
        this.author = author;
        this.copies = copies;
        this.price = price;
    }
}
